src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,30 @@
+package duke;
+
+/**
+ * DukeException implements the Exception that is thrown
+ * in the Duke program.
+ *
+ * @author Alvin Jiang Min Jun
+ * @version v0.1
+ */
+public class DukeException extends Exception{
+
+    /**
+     * DukeException constructor that creates an instance of a DukeException object.
+     *
+     * @param message The message that is contained by the DukeException object.
+     */
+    public DukeException(String message) {
+        super(message);
+    }
+
+    /**
+     * The toString method for the DukeException class.
+     *
+     * @return String The String format of the DukeException Object.
+     */
+    @Override
```

Comment:
is "Duke Exception detected necessary if the Duke Exception already displays an error message?